tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -462,6 +464,52 @@ void ral_qgemm(
   ctx->signalError(Context::FAILURE, "Should turn on bladnn first.");
 }
 
+void ral_comp_intens_fusion(
+    ExecutionContext* ctx,
+    const char* kernel_name,  /* function name to call on host side */
+    const char* dyn_lib_path, /* lib path containing the function */
+    void* stream_handle,      /* stream */
+    void** params /* kernel params */) {
+  void* func_handle = dlopen(dyn_lib_path, RTLD_NOW);
```

Comment:
shall we consider use `ctx->getOrCreateResource` to ensure kernel function is loaded only once and eliminate loading overhead from each invocation.